src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -16046,96 +16035,100 @@ void ComputeTableBodyUsingMovingAvg(EnergyPlusData &state,
                                     Array3D<Real64> const &feneCondInstantSeq,
                                     Array2D<Real64> const &surfDelaySeq)
 {
-    using DataSurfaces::ExternalEnvironment;
-    using DataSurfaces::Ground;
-    using DataSurfaces::GroundFCfactorMethod;
-    using DataSurfaces::KivaFoundation;
-    using DataSurfaces::OtherSideCoefCalcExt;
-    using DataSurfaces::OtherSideCoefNoCalcExt;
-    using DataSurfaces::OtherSideCondModeledExt;
-    using DataSurfaces::SurfaceClass;
-    using General::MovingAvg;
-
-    Array1D<Real64> seqData;     // raw data sequence that has not been averaged yet
-    Array1D<Real64> AvgData;     // sequence data after averaging
+    Array1D<Real64> AvgData;     // sequence data to be averaging
     Array1D<Real64> delayOpaque; // hold values for report for delayed opaque
-    int curExtBoundCond;
-    Real64 singleSurfDelay;
-    auto &ort(state.dataOutRptTab);
-    auto &Zone(state.dataHeatBal->Zone);
-
-    int NumOfTimeStepInDay = state.dataGlobal->NumOfTimeStepInHour * 24;
 
     resultCells = 0.;
     resCellsUsd = false;
     delayOpaque.allocate(LoadCompRow::GrdTot);
-    delayOpaque = 0.;
-    AvgData.allocate(NumOfTimeStepInDay);
-    AvgData = 0.;
+    AvgData.allocate(state.dataGlobal->NumOfTimeStepInHour * 24);
 
     if (desDaySelected != 0 && timeOfMax != 0) {
-
+        // Don't update/average original array data
         // PEOPLE
-        resultCells(LoadCompCol::SensInst, LoadCompRow::People) =
-            MovingAvgAtMaxTime(state, ort->peopleInstantSeq(desDaySelected, _, zoneIndex), NumOfTimeStepInDay, timeOfMax);
+        AvgData = state.dataOutRptTab->peopleInstantSeq(desDaySelected, _, zoneIndex);
+        General::MovingAvg(AvgData, state.dataSize->NumTimeStepsInAvg);
+        resultCells(LoadCompCol::SensInst, LoadCompRow::People) = AvgData(timeOfMax);
```

Comment:
This area avoids acting on the original array, which adds a few lines of code. At this point in the code, could have probably acted directly on the original array but to avoid diffs this method was chosen. This does however do the same thing as done before.